### DISCUSSION TITLE
DateTimeRange header actions not working

### DISCUSSION THREAD

**Discussion by eramosr16** [dotnetcore/BootstrapBlazor#3406] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Inside the Datetime Range  component click on Header Actions aren't working. 

### Expected Behavior

When you click on the Mont/Year is should behave like the DatetimePicker it should allow you to switch between months and years. 

### Steps To Reproduce

To reproduce is very easy go to: https://www.blazor.zone/datetime-range
Click on any of the examples to display the DateTimePicker component, click on Month or Year on the header.

### .NET Version

version 8.04

### Anything else?

![image](https://github.com/dotnetcore/BootstrapBlazor/assets/17813469/f22eb3f2-4f56-4657-bc34-adaf01a38137)

**Comment by bb-auto** [comment]
@eramosr16 Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by eramosr16** [comment]
To reproduce the issue just navigate to the demo page, I already provided steps to reproduce

**Comment by ArgoZhang** [comment]
@eramosr16 hi, this is by design. Prevent the time on the right from being earlier than the time on the left

**Comment by eramosr16** [comment]
I'm referring to  click on the Year and open the year list and select a year and same for the Month selection?.  This is not related to the time validation is just the regular functionality of the DateTimePicker

**Comment by ArgoZhang** [comment]
@eramosr16 I understand your needs, but this is RangePicker and the situation is a bit complicated. Let me think, or you can submit a PR for me to see your implementation ideas

**Reply by ArgoZhang** [reply, marked as answer]
- track issue #3446 

please click the `make as anwser` button
